spec/clang_spec.lua

```diff
@@ -178,4 +177,237 @@ describe("cursor kind", function()
+                end, "calling 'getNumArgs' on bad self (expect cursor with function kind)")
+        end)
+end)
+
+describe("inline function check", function()
+        it("succeeds in identifying an inline function", function()
+                local parser = luaclang.newParser("spec/function_inline.c")
+                local cursor = parser:getCursor()
+                cursor = get_last_child(cursor)
+                assert.are.equals(true, cursor:isFunctionInlined())
+        end)
+
+        it("succeeds in identifying a non-inline function", function()
+                local parser = luaclang.newParser("spec/function.c")
+                local cursor = parser:getCursor()
+                cursor = get_last_child(cursor)
+                assert.are.equals(false, cursor:isFunctionInlined())
+        end)
+
+        it("uses an incompatible cursor with isInlineFunction", function()
+                local parser = luaclang.newParser("spec/struct.c")
+                local cursor = parser:getCursor()
+                cursor = get_last_child(cursor)
+                assert.has.errors(function()
+                        local is_inline = cursor:isFunctionInlined()
+                end, "calling 'isFunctionInlined' on bad self (expect cursor with function kind)")
+        end)
+end)
+
+describe("enum constant value", function()
+        it("succeeds in getting the correct integral value of the EnumConstantDecl", function()
+                local parser = luaclang.newParser("spec/enum.c")
+                local cursor = parser:getCursor()  
+                cursor = get_last_child(cursor)
+                local last_enumconst
+                cursor:visitChildren(function(cursor, parent)
+                     last_enumconst = parent
+                     return "recurse"
+                end)
+                local enum_value = last_enumconst:getEnumValue()
+                assert.are.equal(7, enum_value)
+        end)
+
+        it("uses an incompatible cursor with getEnumValue()", function()
+                local parser = luaclang.newParser("spec/struct.c")
+                local cursor = parser:getCursor()
+                cursor = get_last_child(cursor)
+                assert.has.errors(function()
+                        local enum_value = cursor:getEnumValue()
+                end, "calling 'getEnumValue' on bad self (expect cursor with enum constant kind)")
+        end)
+end)
+
+describe("get storage class", function()
+        it("succeeds in getting the correct storage class speciifer", function()
+                local parser = luaclang.newParser("spec/function.c")
+                local cursor = parser:getCursor()  
+                cursor = get_last_child(cursor)
+                storage_class = cursor:getStorageClass()
+                assert.are.equal("extern", storage_class)
+        end)
+end)
+
+function get_last_member(cursor)
+        cursor:visitChildren(function(cursor, parent)
+                last_member = parent   
```

Comment:
You ended up using a global variable here because you didn't previously declare it.
Add `local last_member` in the previous line.